### DISCUSSION TITLE
I run `npm run tauri android dev` and it keeps showing `Warn Waiting for your frontend dev server to start on http://localhost:14200/...`. How do I set this up?

### DISCUSSION THREAD

**Discussion by llds66** [tauri-apps/tauri#11579] [Q&A]
`PS E:\Code\Demos\tauri-app> npm run tauri android dev

 tauri-app@0.1.0 tauri E:\Code\Demos\tauri-app
tauri "android" "dev"
Info Detected connected device: Medium_Phone_API_35 (sdk_gphone64_x86_64) with target "x86_64-linux-android"
Running BeforeDevCommand (`npm run dev`)

 tauri-app@0.1.0 dev E:\Code\Demos\tauri-app
 vite

(node:26824) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token '??='
    at Loader.moduleStrategy (internal/modules/esm/translators.js:145:18)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:26824) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:26824) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    Warn Waiting for your frontend dev server to start on http://localhost:14200/...
    Warn Waiting for your frontend dev server to start on http://localhost:14200/...
    Warn Waiting for your frontend dev server to start on http://localhost:14200/...
    Warn Waiting for your frontend dev server to start on http://localhost:14200/...
`

**Comment by llds66** [comment]
node version

**Comment by ou-bing** [comment]
@llds66 I have encountered a similar problem, how did you solve it?

**Reply by llds66** [reply]
I replaced the higher version of node to solve it
